src/node-lib/curriculum-api-2023/queries/lessonDownloads/constructLessonDownloads.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  SyntheticUnitvariantLessons,
+  syntheticUnitvariantLessonsSchema,
+} from "@oaknational/oak-curriculum-schema";
+
+import { LessonDownloadsListSchema } from "./lessonDownloads.schema";
+
+import keysToCamelCase from "@/utils/snakeCaseConverter";
+
+const constructLessonDownloads = (
+  downloads: LessonDownloadsListSchema,
```

Comment:
this is a different type to what's passed in from the query - are they actually the same type? if so let's remove one